dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/ArtifactResults.java

```diff
@@ -33,6 +37,7 @@
   private final Map<String, Integer> results = new HashMap<>();
   private final Artifact artifact;
   private String exceptionMessage;
+  private ImmutableListMultimap<DependencyPath, DependencyPath> dependencyTree;
```

Comment:
can we use a more specific type here? i.e. a semantic class instead of a data structure?

Reply:
Will do.

Reply:
This is reverted in favor of `DependencyGraph` class.